meta-resin-common/recipes-support/hostapp-update-hooks/files/0-bootfiles

```diff
@@ -90,16 +90,47 @@ deploy() {
 # MAIN
 #
 
-export DOCKER_HOST="unix:///var/run/docker-host.sock"
+export DOCKER_HOST="unix:///var/run/balena-host.sock"
 
 if [ -z "$container_id" ]; then
 	echo "[ERROR] Container ID provided as an argument is needed."
 	exit 1
 fi
 
-# Deploy all files in the bootfiles list
 bootfiles_list_new="/tmp/$(basename $bootfiles_list).new"
-docker cp "$container_id:$bootfiles_list" "$bootfiles_list_new"
+balena cp "$container_id:$bootfiles_list" "$bootfiles_list_new"
+
+# Do a dry run for copying the boot files and figure out if we would get in a
+# out of space situation
+boot_mnt="$(findmnt --noheadings --canonicalize --output TARGET /dev/disk/by-label/resin-boot)"
+boot_space="$(df -k --output=avail /mnt/boot | grep -v Avail)"
```

Comment:
Can we use bytes everywhere? Choosing 1K as a unit makes things harder

Reply:
Yes. I'll change.